PubSub.DataAccess/SubscriptionRepository.cs

```diff
@@ -0,0 +1,53 @@
+﻿using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using PubSub.Core.Models;
+
+namespace PubSub.DataAccess;
+
+public class SubscriptionRepository : ISubscriptionRepository
+{
+    private readonly PubSubContext _context;
+    public SubscriptionRepository(PubSubContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<IEnumerable<Subscription>> GetAllByUserId(string userId)
+    {
+        return await _context.Subscriptions.Where(x => x.OwnerId == userId).ToListAsync();
```

Comment:
The _context.Subscriptions can be wrapped in an AsQueryable method so this part doesn't need to be repeated with each query 
```
        public IQueryable<Subscription> AsQueryable()
        {
            return _context.Subscriptions.AsQueryable();
        }
```